src/app/pages/settings/containers/settings-page.component.ts

```diff
@@ -212,19 +212,13 @@ export class SettingsPageComponent {
   }
 
   testNotifications() {
+    this.notificationService
+      .sendTestFCMNotification()
+      .then(() => this.firebaseAnalytics.logEvent('notification_test', {}))
     const buttons = [
       {
-        text: this.translate.transform(LocKeys.BTN_CANCEL.toString()),
+        text: this.translate.transform(LocKeys.BTN_OKAY.toString()),
         handler: () => {}
-      },
-      {
-        text: this.translate.transform(LocKeys.CLOSE_APP.toString()),
-        handler: () => {
-          this.notificationService.sendTestFCMNotification().then(() => {
-            this.firebaseAnalytics.logEvent('notification_test', {})
-            this.platform.exitApp()
-          })
-        }
       }
```

Comment:
I would keep the existing functionality even though its possible to receive in foreground to avoid any changes for the end users (otherwise we need to inform them of this well in advance which may not be worth it)